src/main/java/thePackmaster/orbs/contentcreatorpack/Wanderbot.java

```diff
@@ -0,0 +1,109 @@
+package thePackmaster.orbs.contentcreatorpack;
+
+import com.badlogic.gdx.Gdx;
+import com.badlogic.gdx.assets.loaders.TextureLoader;
+import com.badlogic.gdx.graphics.Color;
+import com.badlogic.gdx.graphics.Texture;
+import com.badlogic.gdx.graphics.g2d.SpriteBatch;
+import com.badlogic.gdx.math.MathUtils;
+import com.megacrit.cardcrawl.actions.AbstractGameAction;
+import com.megacrit.cardcrawl.actions.animations.VFXAction;
+import com.megacrit.cardcrawl.actions.common.DamageAction;
+import com.megacrit.cardcrawl.actions.common.DrawCardAction;
+import com.megacrit.cardcrawl.actions.utility.SFXAction;
+import com.megacrit.cardcrawl.cards.DamageInfo;
+import com.megacrit.cardcrawl.cards.DamageInfo.DamageType;
+import com.megacrit.cardcrawl.core.CardCrawlGame;
+import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
+import com.megacrit.cardcrawl.helpers.ImageMaster;
+import com.megacrit.cardcrawl.localization.OrbStrings;
+import com.megacrit.cardcrawl.monsters.AbstractMonster;
+import com.megacrit.cardcrawl.orbs.AbstractOrb;
+import com.megacrit.cardcrawl.vfx.combat.DarkOrbActivateEffect;
+import com.megacrit.cardcrawl.vfx.combat.LightningOrbPassiveEffect;
+import com.megacrit.cardcrawl.vfx.combat.SmallLaserEffect;
+import thePackmaster.cards.highenergypack.StruckByATrain;
+import thePackmaster.util.ImageHelper;
+import thePackmaster.util.TexLoader;
+
+import static thePackmaster.util.Wiz.atb;
+
+public class Wanderbot extends AbstractOrb {
+    private static final OrbStrings orbString;
+    private static final Texture myTex = TexLoader.getTexture("anniv5Resources/images/orbs/wanderbot_orb.png");
+
+    public Wanderbot() {
+        this.ID = "Wanderbot";
```

Comment:
This should use `makeID`.

While you're fixing this, I noticed that Ghostflame (which perhaps you copied this from?) also doesn't use makeID. Could you fix that as well?

Reply:
Copy that, done!